docs/source/conf.py

```diff
@@ -39,8 +49,11 @@
 extensions = [
     "sphinx.ext.autodoc",
     "sphinx.ext.napoleon",
+    "nbsphinx",
 ]
 
+source_suffix = [".rst", ".ipynb"]
```

Comment:
I think we should set `nbsphinx_execute = 'always'` here (cf https://nbsphinx.readthedocs.io/en/0.8.1/never-execute.html) to ensure that the rendered documentation is always up to date with the current algorithms implementation.

Reply:
Done!

Reply:
This triggered an error with CI building documentation saying `Notebook error:
NoSuchKernel in read-only-examples/FeatureSelection.ipynb:
No such kernel named python3`

Do we need to specify the kernel as well in `conf.py`?

Reply:
I have no idea, but it might be related to the issue we are seeing when running examples as tests (#68, one potential root cause is that the kernel is not installed).

> Do we need to specify the kernel as well in conf.py?

nbsphinx doc (https://nbsphinx.readthedocs.io/en/0.8.2/) does not mention it AFAIK, maybe we could look through their issues or open a new one.

Reply:
Wait, they do mention it here: https://nbsphinx.readthedocs.io/en/0.8.2/installation.html#Jupyter-Kernel, and suggest adding `ipykernel` to the requirements

Reply:
Let's see if that works!